### ISSUE TITLE
Map list is capped in UI

### ISSUE THREAD

**Issue by Ephreaym** [opennox/opennox#651]
Would be nice to expand the list to include all installed maps.

**Comment by dennwc** [comment]
@Ephreaym could you elaborate on this one?

I've seen the message on Discord, and it does show only 6 CTF maps. But I'm pretty sure UI can show a ton of Arena maps.

So is it CTF-specific? Or do you see a short list for Arena too?

**Comment by Ephreaym** [comment]
No, I don’t think it’s gamemode specific. It goes for arena as well. I think it only shows an X amount of total maps, and might be capped on amount of maps with a starting letter as well.
